modules/internal/String.chpl

```diff
@@ -821,6 +821,12 @@ module String {
       */
     inline proc size return numCodepoints;
 
+    /*
+      :returns: The indices that can be used to index into the string
+                (i.e., the range `1..size`)
```

Comment:
I am not sure if you intended for this to be inline code (like `1..size`) but if you did, recall that rst requires 2 back tics:

```suggestion
                (i.e., the range ``1..size``)
```

From what I can tell, this is the formatting other string methods use in their doc strings, e.g. [item](https://chapel-lang.org/docs/master/builtins/String.html#String.string.item)

Reply:
(This applies to all doc-strings added in this PR)

Reply:
Sigh... so many markup formats, so few brain cells to store them in.